packages/mock-server/src/api/resolvers.ts

```diff
@@ -95,6 +97,59 @@ const createListResponse = <T>(
 });
 
 export const ResolverService = {
+  requisition: {
+    get: {
+      byId: (id: string): ResolvedRequisition => {
```

Comment:
I'm not seeing what's wrong - but I'm getting undefined for the `id` if I try to query a specific requisition 🤷

Reply:
Ah yes I see the issue, sorry - I'll fix that in a later PR